.github/workflows/ci.yml

```diff
@@ -182,14 +174,12 @@ jobs:
           nkg_tag=$(echo ${{ steps.meta.outputs.tags }} | cut -d ":" -f 2)
           make update-nkg-manifest NKG_PREFIX=${nkg_prefix} NKG_TAG=${nkg_tag}
         working-directory: ./conformance
-    
-      - name: Build binary
-        uses: goreleaser/goreleaser-action@336e29918d653399e599bfca99fadc1d7ffbc9f7 # v4.3.0
+
+      - name: Fetch Cached Artifacts
```

Comment:
The binary needs to be rebuilt here because the previous step updates the manifest with the image being tested, and that manifest is built into the binary. Otherwise the correct image will not be used in the conformance tests.

Reply:
fixed it 👍